.circleci/config.yml

```
@@ -87,6 +87,10 @@ reusable-steps:
     run:
       name: Update Composer
       command: sudo composer self-update
+  - &require-doctrine-odm
```

```suggestion
  - &install-doctrine-mongodb-odm
```

Also, please keep the reusable steps in alphabetical order.